app/services/model/RejectionReason.scala

```diff
@@ -25,7 +25,7 @@ import play.api.libs.json.{Json, OFormat}
 
 import scala.io.Source
 
-case class RejectionReason(code: String, cdsDescription: String, exportsDescription: String, pointer: Option[Pointer])
+case class RejectionReason(code: String, cdsDescription: String, exportsDescription: String, url: Option[String], pointer: Option[Pointer])
```

Comment:
Where are these url templates coming from?  I was expecting to see them defined in a config file our something similar.

Reply:
`conf/code-lists/errors-dms-rej-list.csv`

From this file.

https://github.com/hmrc/customs-declare-exports-frontend/pull/884/files#diff-6a90a36baf2cf59e5e17a114f2cd9261R109

Reply:
Yes - I missed that in the massive diff.

Reply:
Hence my question about the commas.

Reply:
We need to move to `Json`, but this ticket is a chance to try and get design/product feedback ASAP.
Probably move a bit of this logic to a separate microservice... again... lets wait for the feedback from stakeholders before committing ourselves.